recipes/_redhat_client.rb

```diff
@@ -23,5 +23,5 @@
 # To force removing of mariadb-libs on CentOS >= 7
 package 'mysql-libs' do
   action :remove
-  not_if { use_os_native }
+  not_if { use_os_native || node['platform_version'].to_i >= 7 }
```

Comment:
Do you know if the same problem is experienced when using RHEL? 

Maybe it's safer to replace this with the below for now?

```
not_if { use_os_native || (node['platform'] == 'centos' && node['platform_version'].to_i >= 7) }
```

That way if a RHEL user reports the same issue we can just remove the `node['platform']` reference, thoughts?